### PR TITLE
Add blog for Stargate v2 Design

### PR DESCRIPTION
**Pull request by dougwettlaufer**

### CODE REVIEW

File: _posts/2021-11-02-introducing-the-design-for-stargate-v2.md  [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+date:   2021-11-02 00:00:00 -0600
+synopsis: "When we announced Stargate v2, we mainly focused on the \"why\" behind the change. In this post, we’ll dive into the \"how.\""
+author_info:
+  name:
+    first: "Doug"
+    last: "Wettlaufer"
+  picture: "assets/images/stargate-profile.png"
+---
+
+When we [announced Stargate v2](https://stargate.io/2021/11/01/announcing-stargate-v2.html), we mainly focused on the "why" behind the change. In this post, we’ll dive into the "how."
```
Comment (jeffreyscarpenter) [inline _posts/2021-11-02-introducing-the-design-for-stargate-v2.md ]: isn't the actual link https://stargate.io/2021/11/02/announcing-stargate-v2.html

File: _posts/2021-11-02-introducing-the-design-for-stargate-v2.md  [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+Furthermore, this approach would require creating a new schema definition language to transport the requests between services. Although this would certainly be helpful for services that operate at a higher level than plain CQL (like the Documents API), we thought it was unnecessarily complex.
+
+
+### Design 2: Using CQL to communicate
+
+The next option we considered was similar to the “shared nothing” design, but differed in that it used CQL as the communication protocol.
+
+
+![](/assets/images/stargate-v2-cql-shared-nothing.png)
```
Comment (jeffreyscarpenter) [inline _posts/2021-11-02-introducing-the-design-for-stargate-v2.md ]: Did you mean to have a figure label for this as you have for other images?

File: _posts/2021-11-02-introducing-the-design-for-stargate-v2.md  [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+
+## Final notes
+
+We believe that the changes proposed in this design will lead to a much better experience for those who contribute to the Stargate project and those who run it to support their other applications. 
+
+Contributors will be able to quickly iterate on just the parts of the project they want — without needing to understand as deeply the other components. Plus, it’ll open up the potential for newer Java versions and frameworks rather than Java 8 with OSGi or even a polyglot environment. Meanwhile, operators will have greater flexibility over their Stargate deployment and the potential for a lower-resource footprint when scaling to handle higher loads. 
+
+Overall, we’re excited about these new changes and look forward to working with the [Stargate community](https://dtsx.io/3pMAp0Y) to make this new design a reality. On that note, if you want to review this design and give us your feedback, [join the discussion on GitHub](https://github.com/stargate/stargate/discussions/1381). \
```
Comment (jeffreyscarpenter) [inline _posts/2021-11-02-introducing-the-design-for-stargate-v2.md ]: Remove trailing backslash on this line?

**Review by jeffreyscarpenter — COMMENTED** [review]
the filename has a space at the end of it "_posts/2021-11-02-introducing-the-design-for-stargate-v2.md "

File: _posts/2021-11-02-introducing-the-design-for-stargate-v2.md  (hunk 1 above)
Comment (dougwettlaufer) [reply _posts/2021-11-02-introducing-the-design-for-stargate-v2.md ]: Yes, good catch.

File: _posts/2021-11-02-introducing-the-design-for-stargate-v2.md  (hunk 2 above)
Comment (dougwettlaufer) [reply _posts/2021-11-02-introducing-the-design-for-stargate-v2.md ]: Yes, fixed.

File: _posts/2021-11-02-introducing-the-design-for-stargate-v2.md  (hunk 3 above)
Comment (dougwettlaufer) [reply _posts/2021-11-02-introducing-the-design-for-stargate-v2.md ]: Missed that one. Not sure why the docs -> markdown randomly adds those.

**Comment by dougwettlaufer** [comment]
@jeffreyscarpenter where are you seeing the space? Locally it looks like the filename is fine.

**Pull request merged**